Remove npm-check dependency

jzolnowski opened the pull request:
Fix the [Got allows a redirect to a UNIX socket](https://github.com/xlts-dev/angularjs-angular-hybrid-angular-cli/security/dependabot/60) security issue

Commit:
chore(package): replace npm-check with npm-check-updates
- remove `npm-check` due to
  - [got allows a redirect to a unix socket](https://github.com/xlts-dev/angularjs-angular-hybrid-angular-cli/security/dependabot/60) issue
  - lack of regular maintenance
- add [npm-check-updates](https://github.com/raineorshine/npm-check-updates)[npm-check-updates](https://github.com/raineorshine/npm-check-updates)
- update 'update' npm script

gkalpak commented on package-lock.json:
We shouldn't be changing to lockfile version in this PR.

jzolnowski commented:
I updated the commit message, generated package-json without changing `lockfileVersion`, replaced `npm-check` with [npm-check-updates](https://github.com/raineorshine/npm-check-updates). At this point, it doesn't have any [vulnerabilities](https://security.snyk.io/package/npm/npm-check-updates)

gkalpak reviewed (approved):
The changes lgtm :+1:
